user_docs/en/changes.t2t

```diff
@@ -4,21 +4,26 @@ What's New in NVDA
 %!includeconf: ../changes.t2tconf
 
 = 2021.2 =
+This release introduces preliminary Windows 11 support.
+While Windows 11 is yet to be released, this release has been tested on preview versions of Windows 11.
+This includes an important fix for Screen Curtain (see important notes).
+The COM Registration Fixing Tool can now resolve more problems when running NVDA.
+There are updates to the synthesizer eSpeak and braille translator LibLouis.
+There are also various bug fixes and improvements, notably for braille support and Windows terminals, calculator, emoji panel and clipboard history.
+
 
 == Important notes ==
-- Screen Curtain on previous releases of NVDA is no longer compatible with versions of Windows newer than Windows 10 21H2 (10.0.19044).
+Screen Curtain is no longer compatible when using versions of NVDA older than 2021.2 with Windows 10 21H2 (10.0.19044) or later.
 This includes Windows 10 Insiders and Windows 11.
 This is due to a change in the Windows Magnification API.
-When using previous NVDA releases and Windows newer than Windows 10 21H2, screen curtain may fail to work without warning.
-To ensure security, confirm that screen curtain works properly with new or experimental versions of Windows.
+For security purposes, get visual confirmation that the Screen Curtain works properly when using a new version of Windows for the first time.

```

Comment:
Maybe instead of "works properly", "makes the screen entirely black"
```suggestion
For security purposes, when using a new version of Windows, get visual confirmation that the Screen Curtain makes the screen entirely black.
```